### ISSUE TITLE
Add new "conda-incubator" channel for incubator projects

### ISSUE THREAD

**Issue by travishathaway** [conda/infrastructure#954]
### Checklist

- [X] I added a descriptive title
- [X] I searched open requests and couldn't find a duplicate

### What is the idea?

I currently post canary builds to my personal anaconda.org account ("thath"). I would rather not do this and also don't want to create canary channels for each project (e.g. conda-protect, conda-auth, etc.). Instead it would be nice to have a shared "conda-incubator" channel available that I could use for this purpose.

### Why is this needed?

Having a shared channel like this would also help the handover if there ever comes a time where I am not the primary maintainer of my projects in conda-incubator.

### What should happen?

Create a new "conda-incubator" channel on anaconda.org and add myself and others to it so we can begin uploading our canary builds for our projects.

**Comment by kenodegard** [comment]
Makes sense, probably do not want canaries for unofficial projects pushed to `conda-canary`

**Comment by jezdez** [comment]
Wondering if it would suffice to call it `conda-incubator` though, since I don't think we should force a "canary" type build pattern to incubator projects and they could use the channel for whatever pattern they deem useful.
